ui/app/AppLayouts/Communities/views/OwnerTokenWelcomeView.qml

```diff
@@ -0,0 +1,198 @@
+        }
+
+        InfoPanel {
+            isOwner: true
+            checkersModel: [
+                qsTr("Only 1 will ever exist"),
+                qsTr("Hodler is the owner of the Community"),
+                qsTr("Ability to airdrop / destroy TokenMaster token"),
+                qsTr("Ability to mint and airdrop Community tokens")
+            ]
+        }
+
+        InfoPanel {
+            isOwner: false
+            showTag: true
+            checkersModel: [
+                qsTr("Unlimited supply"),
+                qsTr("Grants full Community admin rights"),
+                qsTr("Ability to mint and airdrop Community tokens"),
+                qsTr("Non-transferrable"),
+                qsTr("Remotely destructible by the Owner token hodler")
+            ]
+        }
+
+        component InfoPanel : Rectangle {
+            id: panel
+
+            property bool isOwner
+            property bool showTag
+            property alias checkersModel: checkersItems.model
+            readonly property int margins: 24
+
+            Layout.fillWidth:  true
+            Layout.preferredHeight: panelRow.implicitHeight
+
+            color: "transparent"
+            radius: 8
+            border.color: Theme.palette.baseColor2
+            clip: true
+
+            RowLayout {
+                id: panelRow
+
+                width: parent.width
+                spacing: panel.margins
+
+                PrivilegedTokenArtworkPanel {
+                    Layout.alignment: Qt.AlignTop
+                    Layout.topMargin: panel.margins
+                    Layout.leftMargin: Layout.topMargin
+
+                    isOwner: panel.isOwner
+                    artwork: root.communityLogo
+                    color: root.communityColor
+                    showTag: panel.showTag
+                }
+
+                ColumnLayout {
+                    id: col
+
+                    Layout.alignment: Qt.AlignTop
+                    Layout.topMargin: panel.margins
+                    Layout.bottomMargin: panel.margins
+                    Layout.fillWidth: true
+
+                    Item {
+                        id: panelTextHeader
+
+                        Layout.fillWidth: true
+                        Layout.preferredHeight: headerRow.implicitHeight
+                        Layout.rightMargin: panel.margins
+
+                        RowLayout {
+                            id: headerRow
+
+                            spacing: 8
+
+                            StatusBaseText {
+                                Layout.alignment: Qt.AlignBottom
+                                Layout.maximumWidth: panelTextHeader.width - symbol.width
```

Comment:
WDYT of removing the `panelTextHeader` wrapper and use `Layout.fillWidth: true` instead? I'm missing the purpose of the `Item` wrapper. If removing it would allow us to also remove the manual width calculation here, that would be awesome.

Reply:
The propose for this is to easily calculate the maximum width of the title text since I want it to be elide, but the symbol text being aligned to the title right always. 

If I remove the `Item`, I need to add and `Item filler` to have this alignement correctly displayed and the maximum widh calculation will be more or less the same thing.

So an "extra" item here seems to be needed anyway although you see a different approach that I would kindly appreciate!! :)

Reply:
Got it now! IMO your implementation is the best way to achieve this.
Thanks for the explanation 🍻